apps/sync/test/unit/sync-engine/adapter-mock.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+/* exported
+  AdapterMock
+*/
+
+var AdapterMock = (action, args) => {
+  return {
+    update(kintoColl) {
```

Comment:
Are you referring to the use of ES6 functions object-member shorthand? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#Description